src/storage.ts

```diff
@@ -1,11 +1,17 @@
-const BASE_DIR = 'kv'
-
-export const set = (key: string, value: any) => {
-  // writeFileSync(BASE_DIR + '/' + key, JSON.stringify(value))
-  return true
+export const set = async (
+  KV: KVNamespace,
+  context: ExecutionContext,
+  key: string,
+  value: any
+) => {
+  if (value) {
+    await KV.put(key, value)
+    return true
+  }
+  return false
 }
 
-export const get = (key: string) => {
-  // return JSON.parse(readFileSync(BASE_DIR + '/' + key).toString())
-  return ''
+export const get = async (KV: KVNamespace, key: string) => {
+  const value = await KV.get(key)
+  return value || undefined
```

Comment:
wondering - can we just do `return await KV.get(key)` ?

Reply:
@bjesus ah, yes, actually we can, just need to change the manager.get return type as well, because namespace.get returns string | null